### ISSUE TITLE
why set to float?What will happen if using the default dtype?

### ISSUE THREAD

**Issue by Liyw979** [krumo/Domain-Adaptive-Faster-RCNN-PyTorch#11]
https://github.com/krumo/Domain-Adaptive-Faster-RCNN-PyTorch/blob/276fcf8ae41be483db94682722c8093703750fdd/maskrcnn_benchmark/modeling/da_heads/loss.py#L70-L72

https://github.com/krumo/Domain-Adaptive-Faster-RCNN-PyTorch/blob/276fcf8ae41be483db94682722c8093703750fdd/maskrcnn_benchmark/modeling/da_heads/loss.py#L86-L88

**Comment by krumo** [comment]
Hi, you could try removing this type conversion yourself and see what would happen. If my memory is correct, F.binary_cross_entropy_with_logits() requires a float type input otherwise it will throw an error. Please fix me if I am wrong.

**Comment by Liyw979** [comment]
I use the default dtype which works fine.Maybe this function has been updated.
Thanks for you reply
